db_handle/manage_portfolio.py

```diff
@@ -0,0 +1,112 @@
+import sys
+sys.path.append(r'.')
+from db_handle import postgres_conn
+import random, string
+
+
+"""Init connection to the database"""
+postgres_conn.admin_client()
+
+def create_new_category():
+    """Create ranomly generated category ID"""
+    category_id = []
+    for i in range(1, 7):
+        category_id.append(random.choice(string.digits))
+    category_id = 'CAT-' + ''.join(category_id)
+
+    """Enter the category name"""
+    category_name = input("Enter category name: ")
+
+    # """Calculates the total sub-categories / for new categories is 0 by default"""
+    # postgres_conn.POSTGRES_CURSOR.execute(f"SELECT COUNT (*) FROM subcategories WHERE parent_category = '{category_name}';")
+    # sub_categories_number = postgres_conn.POSTGRES_CURSOR.fetchone()
+    #
+    # """Calculates the total products inside the category / for new categories is 0 by default"""
+    # postgres_conn.POSTGRES_CURSOR.execute(f"SELECT COUNT (*) FROM products WHERE category = '{category_name}';")
+    # products_number = postgres_conn.POSTGRES_CURSOR.fetchone()
+    
+    """Save the category into the database"""
+    postgres_conn.POSTGRES_CURSOR.execute(f"INSERT INTO categories VALUES ('{category_id}', '{category_name}', '{0}', '{0}')")
+    postgres_conn.POSTGRES_CONNECTION.commit()
+
+def create_new_subcategory():
+    """Create ranomly generated sub-category ID"""
+    subcategory_id = []
+    for i in range(1, 7):
+        subcategory_id.append(random.choice(string.digits))
+    subcategory_id = 'SUBCAT-' + ''.join(subcategory_id)
+    
+    """Enter the sub-category name"""
+    subcategory_name = input("Enter the new subcategory name: ")
+
+    # """Calculates the total products inside the sub-category / for new sub-categories is 0 by default"""
```

Comment:
@rayapetkova We won't use this section with the comments. Could you remove this block of code from the function.

Reply:
Should I remove only row 42 or remove all four - 39, 40, 41 and 42?

Reply:
Rows from 42 to 44, only the currently commented lines. Same for above function `create_new_category`